### PR TITLE
feature(website): mobile app download page

### PR DESCRIPTION
**Pull request by ssandino**
Created a simple page for recipients to download the app for their smartphone.

### CODE REVIEW

**Comment by ssandino** [comment]
This PR addresses our app's difficulty being found in the Play Store due to its generic name, low installation numbers, or policy issues with the former version (meanwhile fixed). It allows us to provide a single page to recipients for easy access. @mkue thanks for reviewing and releasing.

File: website/src/app/[lang]/[region]/(website)/app/page.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+						>
+							<Image src={Playstore} width={150} height={50} alt="Playstore Button" />
+						</a>
+					</div>
+					<div className="m-2 flex flex-col items-center justify-center p-4 md:w-1/2">
+						<Typography as="h2" size="xl" className="pb-4">
+							{translator.t('apple')}
+						</Typography>
+						<a
+							href="https://apps.apple.com/ch/app/social-income/id6444860109"
```
Comment (KBEMobisys) [inline website/src/app/[lang]/[region]/(website)/app/page.tsx:43]: Use a link without country code 'CH' like https://apps.apple.com/app/social-income/id6444860109

[the remark this answers is not in the document]
Comment (ssandino) [reply website/src/app/[lang]/[region]/(website)/app/page.tsx:43]: Very good point, missed that one. I'll change it

**Review by KarinBerg — APPROVED** [review]

File: website/src/app/[lang]/[region]/(website)/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+					{translator.t('subtitle')}
+				</Typography>
+			</div>
+			<div className="container mx-auto pb-24 pt-1 md:pt-6">
+				<div className="flex flex-col md:flex-row">
+					<div className="m-2 flex flex-col items-center justify-center p-4 md:w-1/2">
+						<Typography as="h2" size="xl" className="pb-4">
+							{translator.t('android')}
+						</Typography>
+						<a
```
Comment (mkue) [inline website/src/app/[lang]/[region]/(website)/app/page.tsx:30]: Please use the `Link` component instead of `a`

Comment (mkue) [reply website/src/app/[lang]/[region]/(website)/app/page.tsx:30]:
```
import Link from 'next/link';
```

Comment (ssandino) [reply website/src/app/[lang]/[region]/(website)/app/page.tsx]: done

**Review by mkue — APPROVED** [review]

**Pull request merged**